src/main/java/formflow/library/controllers/FormFlowErrorController.java

```diff
@@ -0,0 +1,88 @@
+package formflow.library.controllers;
+
+import static org.springframework.boot.web.error.ErrorAttributeOptions.Include.BINDING_ERRORS;
+import static org.springframework.boot.web.error.ErrorAttributeOptions.Include.EXCEPTION;
+import static org.springframework.boot.web.error.ErrorAttributeOptions.Include.MESSAGE;
+import static org.springframework.boot.web.error.ErrorAttributeOptions.Include.STACK_TRACE;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Arrays;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.error.ErrorAttributeOptions;
+import org.springframework.boot.web.servlet.error.ErrorAttributes;
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class FormFlowErrorController implements ErrorController {
+
+  private final boolean showStackTrace;
+  private final String errorPath;
+
+  private final String prettyPrintPackages;
+  private final ErrorAttributes errorAttributes;
+
+  public FormFlowErrorController(
+      @Value("${form-flow.error.show-stack-trace:false}") boolean showStackTrace,
+      @Value("${server.error.path:${error.path:/error}}") String errorPath,
+      @Value("${form-flow.error.pretty-print-packages:formflow}") String prettyPrintPackages,
+      ErrorAttributes errorAttributes) {
+    this.showStackTrace = showStackTrace;
+    this.errorPath = errorPath;
+    this.errorAttributes = errorAttributes;
+    this.prettyPrintPackages = prettyPrintPackages;
+  }
+
+  @RequestMapping("/error")
+  public ModelAndView handleError(WebRequest webRequest) {
+    String prettyException = prettyPrintStackTrace(errorAttributes.getError(webRequest));
+
+    Map<String, Object> errorAttributesMap = errorAttributes.getErrorAttributes(
+        webRequest,
+        ErrorAttributeOptions.defaults().including(BINDING_ERRORS, EXCEPTION, MESSAGE, STACK_TRACE)
+    );
+    errorAttributesMap.put("prettyStackTrace", prettyException);
+    if (showStackTrace) {
```

Comment:
When I create a thymeleaf error and turn `showStackTrace` off, I assume it shows the apache error screen? Which does include a stack trace. With the name, I was assuming there wouldn't be a stack trace on the page.

Maybe that's just a name expectation I'm having? Should the variable name be updated or is this the expected behavior?

<img width="1325" alt="Screenshot 2023-06-02 at 10 44 23 AM" src="https://github.com/codeforamerica/form-flow/assets/9101728/9a0998ed-d320-4ffb-ad70-4f1b3e5b9262">